jupyterhub/values.schema.yaml

```diff
@@ -1410,6 +1410,31 @@ properties:
               An alias for api_token provided for backward compatibility by
               the JupyterHub Helm chart that will be transformed to
               api_token.
+      customScopes:
+        type: object
+        additionalProperties: true
+        description: |
+          This is where you should define JupyterHub custom scopes and apply them to
+          JupyterHub users, groups, and services to grant them additional
+          permissions as defined in JupyterHub's RBAC system.
+
+          Complement this documentation with [JupyterHub's
+          documentation](https://jupyterhub.readthedocs.io/en/stable/rbac/scopes.html#custom-scopes)
+          about `custom_scopes`.
+
+          ```yaml
+          hub:
+            customScopes:
+              "custom:myservice:read":
+                description: read-only access to myservice
+
+              "custom:myservice:write":
+                description: write access to myservice 
+                subscopes: ["custom:myservice:read"]
+          ```
+
+          When configuring JupyterHub custom scopes via this Helm chart, the `name`
+          field can be omitted as it can be implied by the dictionary key.
```

Comment:
There doesn't seem to be a `name` field for custom_scopes, instead its the dictionary key. So while `JupyterHub.load_roles` is a list and `hub.loadRoles` is a dict translated to a list, the `JupyterHub.custom_scopes` is a dict and doesn't need a translation.

This also means that we can without issues configure this directly via `hub.config.JupyterHub.custom_scopes` as well without providing a dedicated option for this I think.

@Ph0tonic could you update this description to not mention the `name` field?